moveit_ros/perception/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -280,7 +286,12 @@ bool OccupancyMapMonitor::getShapeTransformCache(std::size_t index, const std::s
         std::map<ShapeHandle, ShapeHandle>::const_iterator jt = mesh_handles_[index].find(it->first);
         if (jt == mesh_handles_[index].end())
         {
-          ROS_ERROR_THROTTLE(1, "Incorrect mapping of mesh handles");
+          // ROS_ERROR_THROTTLE(1, "Incorrect mapping of mesh handles");
+          //print every sec, something similar to ROS_ERROR_THROTTLE, there is nothing similar on ROS2
+          while (true) {
```

Comment:
Careful! You cannot used a infinity loop here! You are blocking the program here

Reply:
Yes, I've changed it to  ``RCUTILS_LOG_ERROR_THROTTLE`` https://github.com/AcutronicRobotics/moveit2/commit/e7f3c1748d8ef863a5e2d5325048ffdd8f84d0c6